doc/environment.yml

```diff
@@ -1,9 +1,20 @@
+# documentation dependencies, note that readthedocs uses
+# the environment.lock.yml file, which can be re-generated with
+# the following script:
+# docker run -it --rm continuumio/miniconda3 bash
+# git clone https://github.com/ploomber/jupysql
+# cd jupysql/doc
+# apt update
```

Comment:
is `apt` local installation mandatory ?

Reply:
yes, otherwise conda will include it as part of the environment and we don't need it because the readthedocs image already has it